rilldata/rill
# Remove browser console warnings in Rill Developer

## Checklist

## Summary
#### Details:
The browser console in Rill Developer has been full of warnings about unused props and slots. This PR removes a bunch of those warnings.

## Steps to Verify
1. Do a quick QA of Rill Developer to make sure it runs as expected. Check each page to make sure it doesn't crash.
